src/main/resources/templates/frames/footer.html

```diff
@@ -0,0 +1,5 @@
+<footer class="app-footer">
```

Comment:
I know this was an unplanned collision, but please incorporate my "by-line" template that I created. Of most importance:

* A link to the Github repo for support
* Deploy-time-configurable kudos to the hosting provider -- only Carina so far